src/encoded/tests/data/inserts/gene.json

```diff
@@ -0,0 +1,247 @@
+            "HGNC:ERBAL3",
+            "HGNC:TFCOUP1",
+            "HGNC:COUP-TFI",
+            "HGNC:TCFCOUP1"
+        ],
+        "symbol": "NR2F1\u00c2",
+        "organism": "human",
+        "status": "released",
+        "uuid": "75866336-b42b-4341-9b70-240eb469c231"
+    },
+    {
+        "uuid": "6d4d22fb-99ee-4246-94a0-184aee5d72a1",
+        "dbxref": [
+            "WormBase:WBGene00003222",
+            "WormBase:Y2H9A.1",
+            "UniProtKB:Q9NH52"
+        ],
+        "organism": "celegans",
+        "symbol": "mes-4",
+        "status": "released"
+    },
+    {
+        "uuid": "de7d137a-dafd-46c8-99c3-111b5255e304",
+        "dbxref": [
+            "WormBase:WBGene00003902",
+            "WormBase:Y106G6H.2",
+            "UniProtKB:Q7K797",
+            "UniProtKB:Q9U302",
+            "UniProtKB:Q7K798"
+        ],
+        "organism": "celegans",
+        "symbol": "pab-1",
+        "status": "released"
+    },
+    {
+        "dbxref": [
+           "ENSEMBL:ENSG00000164330",
+            "GeneID:1879",
+            "HGNC:EBF",
+            "HGNC:OLF1",
+            "UniProtKB:Q9UH73"
+        ],
+        "symbol": "EBF1",
+        "organism": "human",
+        "status": "released",
+        "uuid": "cd05d34d-9a51-4efc-b0ec-9245a1d83b40"
+    },
+    {
+        "uuid": "a562a789-e89d-498f-a6e3-3fbbdd042ec3",
+        "symbol": "caudal",
+        "organism": "dmelanogaster",
+        "status": "released"
+    },
+    {
+        "uuid": "08e44197-9c28-4f80-b9ef-2f420869b443",
+        "symbol": "fkh-10",
+        "dbxref": [
+            "WormBase:WBGene00001442",
+            "WormBase:C25A1.2",
+            "UniProtKB:O17593"
+        ],
+        "organism": "celegans",
+        "status": "released"
+    },
+    {
+        "uuid": "4e0994eb-e669-450d-99f4-7e19fc217c0a",
+        "symbol": "FTO",
+        "organism": "human",
+        "dbxref": [
+            "ENSEMBL:ENSG00000140718",
+            "HGNC:24678",
+            "HGNC:ALKBH9",
+            "HGNC:KIAA1752",
+            "HGNC:MGC5149",
+            "GeneID:79068",
+            "UniProtKB:Q9C0B1"
+        ],
+        "status": "released"
+    },
+    {
+        "uuid": "6e42c31d-0fcc-4793-80e1-e93d83f33bb5",
+        "aliases": [],
+        "dbxref": [
+            "FlyBase:CG43860",
+            "FlyBase:FBtr0332562",
+            "FlyBase:FBtr0332558",
+            "FlyBase:CG43860-RA",
+            "FlyBase:FBtr0332557",
+            "UniProtKB:Q9W1P8",
+            "FlyBase:CG43860-RB",
+            "FlyBase:FBtr0332560",
+            "FlyBase:CG43860-RF",
+            "FlyBase:ab-RA",
+            "FlyBase:ab-RD",
+            "RefSeq:NM_079098",
+            "FlyBase:ab-RF",
+            "FlyBase:abrupt",
+            "FlyBase:FBgn0264442",
+            "FlyBase:ab-RB",
+            "FlyBase:CG43860-RD"
+        ],
+        "symbol": "abrupt",
+        "organism": "/organisms/dmelanogaster/",
+        "status": "released"
+    }
+]
```

Comment:
I wonder if we should add some rna genes for testing, like mIR precursors etc.

Reply:
Added 29 non-coding genes in ENCD-3997-modification-1 commit. They cover different categories. It is helpful since I need to update schema (regex for dbxref) accordingly. Thanks.